webapp-src/src/Admin/App.js

```diff
@@ -102,10 +102,10 @@ class App extends Component {
         this.fetchApi();
       } else if (message.type === 'loggedIn') {
         this.setState({loggedIn: message.message}, () => {
-          console.log("loggedIn", this.state.loggedIn);
           if (!this.state.loggedIn) {
-            this.fetchApi();
+            this.setState({curNav: "users",profileList: this.state.profileList.filter((e,i) => i !== 0)});
```

Comment:
This isn't required anymore, the `this.fetchApi();` below will refresh the user list from the backend, and dispatch the new list to all the subcomponents